onnx/defs/schema.h

```diff
@@ -329,7 +351,14 @@ class OpSchema final {
     return *this;
   }
 
-  OpSchema& SetDoc(std::string doc);
+  OpSchema& SetDoc(const std::string& doc) {
```

Comment:
Changed to const-ref. No need to pay for the copy of doc/description if we're not going to copy it. Ditto for Input() and Output() functions.